src/js/page/ui/settings.js

```diff
@@ -33,7 +33,7 @@ export default class Settings {
       this._scroller = document.querySelector('.settings-scroller');
 
       this.container.addEventListener('input', e => this._onChange(e));
-      this._scroller.addEventListener('wheel', e => this._onMouseWheel(e));
+      this._scroller.addEventListener('wheel', e => this._onMouseWheel(e), { passive: true });
```

Comment:
This is also wrong as the code is since `_onMouseWheel` is using `preventDefault` too.

Reply:
But perhaps the Firefox workaround is outdated. We probably need to revisit the code.